dist/cycle-snabbdom.js

```diff
@@ -2947,7 +2947,7 @@ function updateEventListeners(oldVnode, vnode) {
 module.exports = {create: updateEventListeners, update: updateEventListeners};
 
 },{"../is":34}],38:[function(require,module,exports){
-var raf = (typeof window !== `undefined` && window.requestAnimationFrame) || setTimeout;
+var raf = (typeof window !== 'undefined' && window.requestAnimationFrame) || setTimeout;
```

Comment:
For what it's worth, I've noticed Brian tests for undefined a lot via `foo === void 0`, which avoids having to perform a type check. Might be very slightly more efficient.